### PR TITLE
Add new TensorFlow Serving + Flutter text classification codelabs

### PR DESCRIPTION
**Pull request by windmaple**
This PR add the new codelab code for TensorFlow Serving + Flutter. It demos how to call the TensorFlow Serving backend from a Flutter app in the form of a simple spam detection (text classification) use case.

It is broken down into: 
 - `codelab1` contains the first codelab code to train a spam detection model
 - `codelab2` contains the second codelab code to connect Flutter frontend to TensorFlow Serving backend
 - `codelab3` contains the third codelab code to enhance the spam detection model to detect specific spam phrases

### CODE REVIEW

**Comment by windmaple** [comment]
Ubuntu/Windows beta tests are broken due to step_07/integration_test/app_test.dart
Mac beta test is broken due to M1?

Neither is caused by this PR

**Comment by domesticmouse** [comment]
Issue for Mac beta issue: https://github.com/flutter/flutter/issues/100359
Fix for the testing app issue: https://github.com/flutter/codelabs/pull/288 which has it's own issues.

File: flutter_ci_script_stable.sh [hunk 1]
```diff
@@ -18,6 +18,7 @@ declare -a CODELABS=(
   "photos-sharing"
   "star_counter"
   "startup_namer"
+  tfserving-flutter
```
Comment (domesticmouse) [inline flutter_ci_script_stable.sh:21]: nit: surround with " for consistency

**Review by domesticmouse — APPROVED** [review]
LGTM with minor nit

**Comment by domesticmouse** [comment]
Looks like you'll need to exclude the generated code to pass CI: https://github.com/flutter/codelabs/runs/5620448453?check_suite_focus=true#step:7:1548

Docs: https://dart.dev/guides/language/analysis-options#the-analysis-options-file

**Comment by windmaple** [comment]
Thanks! Added fixes.

**Comment by domesticmouse** [comment]
Can you please add a test, if only a smoke test? It helps me confirm things work when I do my quarterly update cycle through the codelabs.

**Comment by domesticmouse** [comment]
> Added basic smoke test. CI passing now on stable.

Woot!

**Pull request merged**